examples/kaas/variables.tf

```diff
@@ -58,3 +58,20 @@ variable "pool_labels" {
     "node-role.kubernetes.io/worker" = "high"
   }
 }
+
+variable "oidc_params" {
+  description = "oidc kubernetes params"
+  type = map(string)
+  default = {
+    "oidc-issuer-url" = "https://your-oidc-url.ch",
+    "oidc-client-id" = "kube-login",
+    "oidc-username-claim" = "email",
+    "oidc-username-prefix" = "oidc:",
+  }
+}
+
+variable "oidc_ca" {
+  description = "oidc CA certificate local file path"
+  type        = string
+  default     = "./oidc_ca.crt"
```

Comment:
issue: the example suggests that the provider will take a filename as a value

suggestion: rename the variable to `oidc_ca_filename` here and add the variable usage in main.tf `oidc_ca = file(var.oidc_ca_filename)`